### PR TITLE
class doc for AttributeGetter and Search.

### PR DESCRIPTION
**Pull request by sharma7n**
Added docstrings for the AttributeGetter class, the Search class, and the NodeBuilder classes in the Search class.

### CODE REVIEW

File: braintree/attribute_getter.py [hunk 1]
```diff
@@ -1,4 +1,18 @@
 class AttributeGetter(object):
+    """
+        Helper class for objects that define their attributes from dictionaries
```
Comment (EvanHahn) [inline braintree/attribute_getter.py]: Would you mind removing the indentation here?

File: braintree/search.py [hunk 2]
```diff
@@ -1,5 +1,12 @@
 class Search:
+	"""
+		Collection of classes used to build search queries.
```
Comment (EvanHahn) [inline braintree/search.py]: Again, would you mind removing the indentation here?

File: braintree/search.py [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+		
+		Each Builder class defines one or more methods that returns a Node
+		object with the name of the field, the comparator, and the value.
+	"""
 	class IsNodeBuilder(object):
+		""" Builds a query for value equality. """
```
Comment (EvanHahn) [inline braintree/search.py]: Could you remove the space padding on either side of this string? Eg, it should look like this:

```py
"""Builds a query for value equality."""
```

**Comment by sdcoffey** [comment]
Hey @sharma7n,

I'm going through some of our open issues here and I wanted to see if this PR is still something you're interested in getting merged? Looks like we just need a few changes and we'll get it merged!

**Comment by sharma7n** [comment]
Hi! Sorry I hadn't responded earlier. I just pushed up the requested changes. Thanks!

**Comment by sdcoffey** [comment]
Thanks again for submitting this PR! Merging now.

**Pull request merged**